e2e/cypress/integration/system_console/ui_and_api/customization_spec.js

```diff
@@ -396,6 +396,34 @@ describe('Customization', () => {
             expect(config.TeamSettings.EnableCustomBrand).to.equal(false);
         });
     });
+
+    it('T1024 - Can change name and desc with Custom Branding set to false', () => {
+        // # Make sure necessary field is false
+        cy.apiUpdateConfig({TeamSettings: {EnableCustomBrand: false}});
+        cy.reload();
+
+        // * Verify that setting is visible and matches text content
+        cy.findByTestId('TeamSettings.SiteNamelabel').scrollIntoView().should('be.visible').and('have.text', 'Site Name:');
+
+        // # Update both Site Name and Description to store test values
+        const siteName = 'Mattermost_Text';
```

Comment:
Nit: did you mean `test`?
```suggestion
        const siteName = 'Mattermost_Test';
```

Reply:
Right! Good catch, thanks! :)